<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="BDC7-L-1Z7XLV1" name="mwo.tests.LoginTests" tests="2" failures="0" timestamp="22 Dec 2016 10:04:32 GMT" time="0.000" errors="2">
  <testcase name="testInValidLogin" time="0.000" classname="mwo.tests.LoginTests">
    <error type="java.lang.IllegalStateException" message="The driver executable does not exist: C:\Users\srinivas.bavirisetti\workspace\IFS\MobileWorkOrder\&quot;C:\Program Files (x86)\Appium\node.exe&quot;">
      <![CDATA[java.lang.IllegalStateException: The driver executable does not exist: C:\Users\srinivas.bavirisetti\workspace\IFS\MobileWorkOrder\"C:\Program Files (x86)\Appium\node.exe"
	at com.google.common.base.Preconditions.checkState(Preconditions.java:199)
	at org.openqa.selenium.remote.service.DriverService.checkExecutable(DriverService.java:121)
	at org.openqa.selenium.remote.service.DriverService$Builder.usingDriverExecutable(DriverService.java:222)
	at io.appium.java_client.service.local.AppiumServiceBuilder.usingDriverExecutable(AppiumServiceBuilder.java:440)
	at com.appium.base.AppiumSingleTest.startAppiumServer(Unknown Source)
	at com.appium.testng.TestBaseStandalone.beforeSuite(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:298)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
    </error>
  </testcase> <!-- testInValidLogin -->
  <testcase name="testValidLogin" time="0.000" classname="mwo.tests.LoginTests">
    <error type="java.lang.IllegalStateException" message="The driver executable does not exist: C:\Users\srinivas.bavirisetti\workspace\IFS\MobileWorkOrder\&quot;C:\Program Files (x86)\Appium\node.exe&quot;">
      <![CDATA[java.lang.IllegalStateException: The driver executable does not exist: C:\Users\srinivas.bavirisetti\workspace\IFS\MobileWorkOrder\"C:\Program Files (x86)\Appium\node.exe"
	at com.google.common.base.Preconditions.checkState(Preconditions.java:199)
	at org.openqa.selenium.remote.service.DriverService.checkExecutable(DriverService.java:121)
	at org.openqa.selenium.remote.service.DriverService$Builder.usingDriverExecutable(DriverService.java:222)
	at io.appium.java_client.service.local.AppiumServiceBuilder.usingDriverExecutable(AppiumServiceBuilder.java:440)
	at com.appium.base.AppiumSingleTest.startAppiumServer(Unknown Source)
	at com.appium.testng.TestBaseStandalone.beforeSuite(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:298)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.TestNG.privateMain(TestNG.java:1355)
	at org.testng.TestNG.main(TestNG.java:1324)
]]>
    </error>
  </testcase> <!-- testValidLogin -->
</testsuite> <!-- mwo.tests.LoginTests -->
